other/io.github.antimicrox.antimicrox.xml

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <mime-info xmlns="http://www.freedesktop.org/standards/shared-mime-info">
   <mime-type type="application/x-amgp">
-    <comment xml:lang="en">AntiMicroX Profile</comment>
+    <comment>AntiMicroX profile</comment>
```

Comment:
Is it possible to later add translations for other languages? Will it work?

For example:
```xml
<comment>AntiMicroX profile</comment>
<comment xml:lang="pl">profile programu AntiMicroX</comment>
```

Reply:
It is: https://specifications.freedesktop.org/shared-mime-info-spec/latest/ar01s02.html#idm44983870071680

> `comment` elements give a human-readable textual description of the MIME type, usually composed of an acronym of the file name extension and a short description, like "ODS spreadsheet". There may be many of these elements with different xml:lang attributes to provide the text in multiple languages. 

With some translation magic like for the appdata file, it's doable, but I don't see this "comment" coming up anywhere.

Reply:
The comment element is used by desktop environments and other graphical programs. The Dolphin file manager for example will display `Testfile.txt (plain text document, 0B)` when hovering over a text file, instead of `Testfile.txt (text/plain, 0B)`.

Reply:
I have checked this way of adding translations. It works in my case. (Ubuntu 20.04 Gnome, Nautilus)